modules/displayBook.js

```diff
@@ -0,0 +1,19 @@
+import deleteFunc from './delete.js';// eslint-disable-line
+import myLib from '../index.js';// eslint-disable-line
+
```

Comment:
- [ ] Same here, You are not allowed to disable the linter, Kindly make sure you fix this instead of disabling it 👍🏻